cmd/infracost/run.go

```diff
@@ -285,7 +301,8 @@ func runProjectConfig(cmd *cobra.Command, runCtx *config.RunContext, ctx *config
 		EnableLogging: runCtx.Config.IsLogging(),
 		NoColor:       runCtx.Config.NoColor,
 	}
-	spinner = ui.NewSpinner("Calculating monthly cost estimate", spinnerOpts)
+	spinner := ui.NewSpinner("Calculating monthly cost estimate", spinnerOpts)
+	defer spinner.Fail()
```

Comment:
do we want `spinner.Fail()` here - or `spinner.Stop()`? This question goes for all the `Fail` method calls.

Reply:
I did `Fail()` because if there's a panic or unexpected error then we show an ❌ . If we've previously called `.Success()` then it won't do anything. I don't think it really matters in this case if the spinner doesn't show that though, so open to change it to `Stop()` if you think that's less confusing.

Reply:
👍  makes sense to me - I was just a bit confused by the method name.